catalog/gitops/source-repositories.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: sourcerepo.cnrm.cloud.google.com/v1beta1
```

Comment:
wouldn't variants in the future be confusing if csr is part of the main `gitops` pkg?

Reply:
I'm thinking CSR is the default and anything else would be a variant.